Ladybird/Qt/BrowserWindow.cpp

```diff
@@ -116,6 +116,12 @@ BrowserWindow::BrowserWindow(Vector<URL::URL> const& initial_urls, WebView::Cook
     edit_menu->addAction(m_select_all_action);
     QObject::connect(m_select_all_action, &QAction::triggered, this, &BrowserWindow::select_all);
 
+    m_paste_action = new QAction("&Paste", this);
```

Comment:
Oh sorry, didn't notice before - I think it makes more sense for "Paste" to go right below "Copy". I don't think I've ever seen a UI that is ordered Copy > Select All > Paste

Reply:
fixed!